test/Microsoft.AspNetCore.SignalR.Tests.Utils/TestConnectionContext.cs

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Protocols;
+using Microsoft.AspNetCore.Protocols.Features;
+
+namespace Microsoft.AspNetCore.Sockets
+{
+    public class TestConnectionContext : ConnectionContext,
```

Comment:
So this is just a copy of `HttpConnectionContext` renamed?

Reply:
And with the ValueStopwatchTimer removed.

Reply:
But is that worth having a near duplicate set of code used for testing?

Reply:
I can move this to common but I don't want it to be a thing. Each "dispatcher" should have their own implementation.